src/PageNotFound.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default () => {
+  return (
+    <div className="router-page router-page-common page-not-found-container">
+      <div>
```

Comment:
really need 2 div's here?)

Reply:
I was centering entire container using flex but then by default child item Home Link button will occupy 100% width of its parent flex container. Therefore I provided extra div. Now I have removed the extra div and added margin auto to link button.